### PR TITLE
feat: added verbose flag and extra output to 'jx status'

### PR DESCRIPTION
**Pull request by garethjevans**
#### Submitter checklist

- [ ] Change is code complete and matches issue description.
- [ ] Change is covered by existing or new tests.

#### Which issue this PR fixes

fixes #

### CODE REVIEW

**Comment by pmuir** [comment]
/test
/hold
Holding PRs as we have a big refactor to get in first

File: pkg/jx/cmd/status.go [hunk 1]
```diff
@@ -49,6 +49,7 @@ func NewCmdStatus(commonOpts *CommonOptions) *cobra.Command {
 	}
 
 	cmd.Flags().StringVarP(&options.node, "node", "n", "", "the named node to get ")
+	options.addCommonFlags(cmd)
```
Comment (ccojocar) [inline pkg/jx/cmd/status.go]: This is not required anymore. The common flags are added in the root command.

**Comment by rawlingsj** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/jenkins-x/jx/pull/3179#issuecomment-465898201" title="LGTM">pmuir</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=jenkins-x%2Fjx).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/jenkins-x/jx/blob/master/OWNERS)~~ [pmuir]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: pkg/jx/cmd/status.go (hunk 1 above)
Comment (jstrachan) [reply pkg/jx/cmd/status.go]: fixed

**Comment by rawlingsj** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/jenkins-x/jx/pull/3179#issuecomment-467303649" title="LGTM">jstrachan</a>*, *<a href="https://github.com/jenkins-x/jx/pull/3179#issuecomment-465898201" title="LGTM">pmuir</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=jenkins-x%2Fjx).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/jenkins-x/jx/blob/master/OWNERS)~~ [jstrachan,pmuir]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by jstrachan** [comment]
@ccojocar I removed that unnecessary `addCommonFlags()` call

**Pull request merged**